.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
HS, but as this is highlighted here: As it's more an application than a library, why would you ignore the `composer.lock` file ?